src/app/services/folder.js

```diff
@@ -412,6 +413,7 @@ module.exports = (Model, App) => {
       where: {
         name: { [Op.eq]: destinationName },
         parent_id: { [Op.eq]: destination },
+        id: { [Op.ne]: folderId }
```

Comment:
we do not support duplicated folders, even with different ids

Reply:
same than file